test/bit_string_test.exs

```diff
@@ -0,0 +1,77 @@
+defmodule BitStringTest do
+  use ExUnit.Case, async: true
```

Comment:
Please move this to `test/query_test.exs` where most other tests are. calendar types have their own test file but they are tricky and we need a way to disable those for old versions of elixr.